### PR TITLE
[DE] Localization.md: minor improvements to German translation

### PR DESCRIPTION
**Pull request by deining**
Correcting grammar in German translation.

### CODE REVIEW

**Comment by rlenferink** [comment]
/uncc
(not enough grammar knowledge)

File: content/de/docs/contribute/localization.md [hunk 1]
```diff
@@ -22,30 +22,30 @@ Da Mitwirkende nicht ihren eigenen Pull Request freigeben können, brauchst du m
 
 Alle Lokalisierungsteams müssen sich mit ihren eigenen Ressourcen selbst tragen. Die Kubernetes-Website ist gerne bereit, deine Arbeit zu beherbergen, aber es liegt an dir, sie zu übersetzen.
 
-### Finden deinen Zwei-Buchstaben-Sprachcode
+### Ermittlung deines Zwei-Buchstaben-Sprachcodes
 
 Rufe den [ISO 639-1 Standard](https://www.loc.gov/standards/iso639-2/php/code_list.php) auf und finde deinen Zwei-Buchstaben-Ländercode zur Lokalisierung. Zum Beispiel ist der Zwei-Buchstaben-Code für Korea `ko`.
 
-### Duplizieren und klonen des Repositories
+### Duplizieren und Klonen des Repositories
```
Comment (sftim) [inline content/de/docs/contribute/localization.md:29]: Why “Klonen”? (Headings should be in sentence case - at least in English; my assumption is that it is the other headings that are wrong).

Comment (deining) [reply content/de/docs/contribute/localization.md:29]: > Why “Klonen”?

Capitalization in German is quite different from English: in German, nouns are always capitalized, regardless of gender, case, or position in the sentence.  And `Klonen` (=cloning) is a noun (as stated [here](https://dict.leo.org/german-english/Klonen)), so my correction is valid. 

Original text: `Fork and clone the repo`.
Literal translation: `Dupliziere und klone das Repository` (imperative).
Better translation: `Duplizieren und Klonen des Repositories` (noun style).

If you decide to follow my second proposal (that's also what the existing translation does), you **definitely** have to write `Klonen` (uppercase). The current wording is simply wrong, as any native German speaker (even with mediocre knowledge of German grammar) will tell you.

**Comment by sftim** [comment]
/uncc
 (not enough grammar knowledge)

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**